server/channels/db/migrations/mysql/000115_create_oauth_outgoing_connection_table.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS OAuthOutgoingConnection (
+    Id varchar(26),
+    Name varchar(64),
+    CreatorId varchar(26) DEFAULT NULL,
+    CreateAt bigint(20) DEFAULT NULL,
+    UpdateAt bigint(20) DEFAULT NULL,
+    ClientId varchar(255),
+    ClientSecret varchar(255),
+    OAuthTokenURL text,
+    GrantType ENUM('client_credentials') DEFAULT 'client_credentials',
```

Comment:
Can you clarify this: this is an enum with just one value? Why is it in the DB then? Do we expect more grant types to be added later? In that case, we should declare a separate enum type to which it can be added.

Reply:
Yes, the idea would be to add more but at this point we are just adding the first one at this point. I declared the enum type in the psql migration, but IIRC mysql doesn't have custom types, does it?  Or do you mean in code?

Reply:
Oops sorry, I didn't see that this is MySQL. Let's remove the enum then. Because altering a column type is a blocking operation.